nautobot_secrets_providers/tests/test_providers.py

```diff
@@ -292,6 +302,24 @@ def test_retrieve_mount_point_success(self, requests_mocker):
         response = self.provider.get_value_for_secret(self.secret_mounting_point)
         self.assertEqual(self.mock_response["data"]["data"]["location"], response)
 
+    @requests_mock.Mocker()
+    def test_retrieve_configuration_success(self, requests_mocker):
```

Comment:
Should we add a `test_retrieve_configuration_non_configured_vault` test as well?